internal/Picker/Picker.module.less

```diff
@@ -95,7 +95,9 @@
 		&.horizontal {
 			.item,
 			.itemDecrement,
-			.itemIncrement {
+			.itemIncrement,
+			.secondaryItemIncrement,
+			.secondaryItemDecrement {
```

Comment:
Could we rename `secondaryItemIncrement` and `secondaryItemDecrement` as `secondaryItem` and use only `secondaryItem` for both secondary increment and decrement item? I think that `secondaryItemIncrement` and `secondaryItemDecrement` have the same rule.

Reply:
good idea 👍

Reply:
Oh, I'm sorry but I think it's not a good idea.
For future reference, I think the previous class names are clearer.
Would you please rollback the change you made here? I think the first commit looks good.

Reply:
@MikyungKim ,I reverted this change.